package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "ripple-lib-extensions",
```

Comment:
That's how it is structured now. Are you intentionally trying to change the structure? Not just gulpfile, but also a separate package.json for each directory.